src/addresses/Address-details/AddressDetails.component.tsx

```diff
@@ -141,7 +135,7 @@ const AddressDetails: FC<AddressDetailsTableProps> = ({ name }) => {
       <Title headingLevel="h2" className="pf-u-mt-md pf-u-ml-md pf-u-mb-md">
         {t('Attributes')}
       </Title>
-      {isSuccess ? (
+      {isSuccess && (
```

Comment:
I think there was a confusion here. You still want to handle the errors of the jolokia calls. It is independent of the rest of the PR, i.e the fact that you are able or not to retrieve the broker CR.